internal/controllers/core/filewatch/controller_test.go

```diff
@@ -348,3 +349,42 @@ func TestController_Disable_Ignores_File_Changes(t *testing.T) {
 	f.MustGet(key, &fwAfterDisable)
 	require.Equal(t, 0, len(fwAfterDisable.Status.FileEvents))
 }
+
+func TestController_IgnoreErrorsOnCancel(t *testing.T) {
```

Comment:
I can be convinced this test should not exist because it's doing some deeply brittle/sketchy things reliant on the internals